CHANGELOG.md

```diff
@@ -2,10 +2,349 @@
+  [#4007](https://github.com/mixxxdj/mixxx/pull/4007)
+  [#4070](https://github.com/mixxxdj/mixxx/pull/4070)
+  [#4084](https://github.com/mixxxdj/mixxx/pull/4084)
+  [#4098](https://github.com/mixxxdj/mixxx/pull/4098)
+  [#4113](https://github.com/mixxxdj/mixxx/pull/4113)
+  [#4163](https://github.com/mixxxdj/mixxx/pull/4163)
+  [#4166](https://github.com/mixxxdj/mixxx/pull/4166)
+  [#4185](https://github.com/mixxxdj/mixxx/pull/4185)
+  [#4187](https://github.com/mixxxdj/mixxx/pull/4187)
+* Drop Ubuntu Bionic support, require Qt 5.12
+  [#3687](https://github.com/mixxxdj/mixxx/pull/3687)
+  [#3735](https://github.com/mixxxdj/mixxx/pull/3735)
+  [#3736](https://github.com/mixxxdj/mixxx/pull/3736)
+  [#3985](https://github.com/mixxxdj/mixxx/pull/3985)
+* Add NixOS support
+  [#2820](https://github.com/mixxxdj/mixxx/pull/2820)
+  [#2828](https://github.com/mixxxdj/mixxx/pull/2828)
+  [#2836](https://github.com/mixxxdj/mixxx/pull/2836)
+  [#2827](https://github.com/mixxxdj/mixxx/pull/2827)
+  [#2827](https://github.com/mixxxdj/mixxx/pull/2827)
+  [#2828](https://github.com/mixxxdj/mixxx/pull/2828)
+  [#3113](https://github.com/mixxxdj/mixxx/pull/3113)
+  [#3089](https://github.com/mixxxdj/mixxx/pull/3089)
+  [#3545](https://github.com/mixxxdj/mixxx/pull/3545)
 * Add Random Track Control to AutoDJ [#3076](https://github.com/mixxxdj/mixxx/pull/3076)
 * Add support for saving loops as hotcues [#2194](https://github.com/mixxxdj/mixxx/pull/2194) [lp:1367159](https://bugs.launchpad.net/mixxx/+bug/1367159)
+* Make use of inclusive language [#2894](https://github.com/mixxxdj/mixxx/pull/2894)
+* Add Noise effect [#2921](https://github.com/mixxxdj/mixxx/pull/2921)
+* Improve Unittests
+  [#2938](https://github.com/mixxxdj/mixxx/pull/2938)
+  [#2980](https://github.com/mixxxdj/mixxx/pull/2980)
+  [#3006](https://github.com/mixxxdj/mixxx/pull/3006)
+* Logging: Add support for QT_MESSAGE_PATTERN env var [#3204](https://github.com/mixxxdj/mixxx/pull/3204)
+* Colored logging console output
+  [#3197](https://github.com/mixxxdj/mixxx/pull/3197)
+* Fix an issue with the `%{function}` placeholder in the QT_MESSAGE_PATTERN for custom logging
+  [#3518](https://github.com/mixxxdj/mixxx/pull/3518)
+* Improve command line argument parser
+  [#3640](https://github.com/mixxxdj/mixxx/pull/3640)
+  [#3962](https://github.com/mixxxdj/mixxx/pull/3962)
+  [#4022](https://github.com/mixxxdj/mixxx/pull/4022)
+  [#4036](https://github.com/mixxxdj/mixxx/pull/4036)
+  [#4170](https://github.com/mixxxdj/mixxx/pull/4170)
+  [#4057](https://github.com/mixxxdj/mixxx/pull/4057)
+* Rename Master EQ to Main EQ for an inclusive language [#3868](https://github.com/mixxxdj/mixxx/pull/3868)
+* Improve message when dealing with macOs sandbox [#4018](https://github.com/mixxxdj/mixxx/pull/4018) [lp:1921541](https://bugs.launchpad.net/mixxx/+bug/1921541)
+* CONTRIBUTING.md: move content from DocuWiki [#2699](https://github.com/mixxxdj/mixxx/pull/2699)
+* Misc. refactorings
```

Comment:
Do we really need to mention that? I guess no user will know what it means.

Reply:
I was unsure about that. We may remove the whole block. For now I have kept it for completeness. 
Ideas?

Reply:
I think the current entry is not really helpful unless you read all the linked PRs. We could split it up (e.g. "Refactor XYZ", "Fix various compiler warnings", etc.), but I guess this is too much without real relevance to the user.

Or maybe something like "And many more fixes, improvements and cleanups. See the 2.4.0 milestone on GitHub for details."?

Reply:
> And many more fixes ..

Without the link is not a good idea, because the user will have hard time to find them among all others already listed above. Should I remove the whole block than, to not make the user curious?